cmd/smarty-usps-comparison/main.go

```diff
@@ -0,0 +1,229 @@
+package main
+
+import (
+	"encoding/csv"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/corbaltcode/usps/internal/smarty"
+)
+
+type ZipcodeResult struct {
```

Comment:
I wasn't sure how valuable it was to keep the mismatchCount field because it's now baked into the errorMessage field. But now I think there are advantages to doing this. It will be more straightforward when analyzing the output programmatically if we don't have to parse the error message string.

Reply:
Yep, and I think it'll also help if you import the output into a spreadsheet. Then you can sort by that column.